WowPacketParser/Misc/ClientVersion.cs

```diff
@@ -256,6 +256,9 @@ public static ClientVersionBuild VersionDefiningBuild
                     case ClientVersionBuild.V6_2_4_21463:
                     case ClientVersionBuild.V6_2_4_21676:
                     case ClientVersionBuild.V6_2_4_21742:
+                    case ClientVersionBuild.V7_0_3_22280:
+                    case ClientVersionBuild.V7_0_3_22289:
+                    case ClientVersionBuild.V7_0_3_22293:
```

Comment:
This will make wpp load 6_0_2_19033.dll for 7.x builds, what I would like to see is loading 7_0_3.dll, then loading 6_0_2 for missing handlers